docs/datafeed.md

```diff
@@ -0,0 +1,127 @@
+# Datafeed
+
+The Datafeed Service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
+The bot can create a datafeed, list all created datafeeds or retrieve all the Real Time events within a datafeed through Datafeed API.
+The Datafeed Service is a Core service built on top of the Datafeed API and provide a dedicated contract to bot developers to work with datafeed. 
+
+## Datafeed Configuration
```

Comment:
"How to use" should goes before configuration section. By default, no configuration is required to make it working